### ISSUE TITLE
Homepage overhaul

### ISSUE THREAD

**Issue by flokuep** [Vocaluxe/Vocaluxe#240]
Before realsing, I would like to give our homepage a complete overhaul.

Currently working on a new page with jekyll, current version: [flokuep.github.io/VocaluxeWebsite](http://flokuep.github.io/VocaluxeWebsite) - Once finished, we'll merge it into [Vocaluxe/VocaluxeWebsite](https://github.com/Vocaluxe/VocaluxeWebsite). I've finished the basic structure of our new page, there are still some tasks to do

- [x] Review all texts
- [x] Add some missing texts and screenshots
- [x] Missing content?
- [x] Find a way to set `JEKYLL_GITHUB_TOKEN` ([Read more](https://github.com/jekyll/github-metadata)) - We need this for automated download-content (currently there is 'no release')
- [x] Maybe some cleanup
- [x] Fix uncorrect unordered list in blogpost
- [x] Lightbox for images

**Comment by derivator** [comment]
👍 for an overhaul. Just wanted to mention the possibility of going even simpler with [Github Pages](https://pages.github.com/). Maybe it would be nice to consolidate and ditch Sourceforge altogether? Github Pages also supports [Jekyll](https://help.github.com/articles/about-github-pages-and-jekyll/) for completely static sites, no php or anything involved. Just a thought.

**Comment by flokuep** [comment]
Yeah. I guess completly switching is a good idea. Currenty don't know who is the owner of vocaluxe-domains to let them redirect to Github Pages (but this should be working sometimes).... @lukeIam has begon some testing with Jekyll and Github Pages. I guess it would be nice to move...

**Comment by lukeIam** [comment]
> Currenty don't know who is the owner of vocaluxe-domains to let them redirect to Github Pages (but this should be working sometimes)....

| Domain       | Owner               | Active    |
|--------------|---------------------|-----------|
| vocaluxe.org | @brunzel            | Yes       |
| vocaluxe.net | @brunzel            | Yes       |
| vocaluxe.de  | Name + Post Address | No (typo) |
| vocaluxe.eu  | @stsundermann       | No        |
| vocaluxe.com | domain trader       | No        |

Even if it takes some time we can still serve a `301 permanently moved` from the sourceforge page and redirect all users.

P.S.: The facebook page uses vocaluxe.de -> so the link there is not working

@all
How should we proceed? Any design ideas?

**Comment by flokuep** [comment]
I've just created a repo for our Homepage and will start working there as suggested with Jekyll. I could try to get into contact with @brunzel canni (I guess he is owner of vocaluxe.de?) and @stsundermann and ask them to change redirection? Facebook page changes are no problem, I'm admin there.

**Comment by lukeIam** [comment]
> I've just created a repo for our Homepage and will start working there as suggested with Jekyll.  

👍 
Just tell me if I can do something. 

Just a little note here: if we want use `https://vocaluxe.github.io` we have the put the page in the branch 
`Vocaluxe/master`. At moment we would get`https://vocaluxe.github.io/VocaluxeWebsite`.  
see https://help.github.com/articles/user-organization-and-project-pages/#user--organization-pages

> I could try to get into contact with @brunzel canni (I guess he is owner of vocaluxe.de?) and @stsundermann and ask them to change redirection?  

If you can contact them that would be great (when we are ready with the new page).

**Comment by brunzel** [comment]
Yes, i'm the owner of vocaluxe.org and vocaluxe.net. But only until 17.01.2017 because i have terminated the contracts.

**Comment by lukeIam** [comment]
@brunzel Would you mind to transfer the domains before that date?
I would takeover one of your two domains (the .org?).

**Comment by brunzel** [comment]
ok, send me a mail (brunzel(at)gmx.li)

**Comment by flokuep** [comment]
I've updated this issue to current status. Would be nice to get some help, I'm on vacation for one week, so no time to work on this task...

**Comment by derivator** [comment]
@flokuep It looks very nice, I'll do some proofreading today.

**Comment by derivator** [comment]
@flokuep @lukeIam oops, I got confused by github's interface and accidentally already pushed flokuep's changes to Vocaluxe/VocaluxeWebsite instead of my own repository... I could force push to undo, but since nothing links there yet we might as well leave it, right?

**Comment by flokuep** [comment]
I think it's okay :)

**Comment by lukeIam** [comment]
fighting with the error: `Liquid Exception: Error:  SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B` 
when building local (is there somewhere a cert configured?)

**Comment by derivator** [comment]
I made [a few improvements](https://github.com/derivator/VocaluxeWebsite/commits/master) (fixed typos and changed/added some descriptions), preview [here](https://derivator.github.io/VocaluxeWebsite/).

**Comment by flokuep** [comment]
Nice, thanks!

**Comment by lukeIam** [comment]
@flokuep nice work!
I also made some changes in my fork:
https://github.com/lukeIam/VocaluxeWebsite-1

Some general things:
- lightbox/fancebox for images in articles would be nice (or make them bigger by default if there is space)
- I wouldn't show the source code zips in the download section (those who need them will find them)
- do we want to "steal" some of the CC songs from the USDX homepage and link them directly into our song section?

**Comment by flokuep** [comment]
Thanks :) Lightbox would be nice, I'll add this as todo. Maybe we just could link the cc-songs?

We now need to merge all changes :D I guess best way would be to open pull request to Vocaluxe/VocaluxeWebsite?

**Comment by derivator** [comment]
> Nice, thanks! Do we want to add a comment function like disqus?

Imo, considering the comments on the old site, at this point it's not worth the effort. If people want to report bugs or get help with problems, they should use the issue tracker/forum, adding another thing will only dilute our attention. And the old site only ever had two posts, so if we don't post *a lot* more often to the new site, we don't need comments... If we do end up posting more often, we can still add them later.

**Comment by lukeIam** [comment]
> Do we want to add a comment function like disqus?

Comments on the page would an other thing to monitor (beside facebook, github and the forum)
(As disqus is blocked by my Ghostery addon - I haven't used it yet 😉 )

An other thing in the same direction: 
visitor statistics (the old user used piwik) but as the site is now static we can't host a piwik installation there.
- do we want to use piwik again?
- where to host? I played a bit with piwik on Azure and with MySQL in-app configuration it looks quite usable with my free plan. (Free ClearDB MySql instances are slow as hell)

**Comment by flokuep** [comment]
Okay. No comments 😃 - Quite don't know if we really need piwic or visitor statistics anyway?

**Comment by lukeIam** [comment]
> Maybe we just could link the cc-songs?

Their songs are also from the forum:
http://forum.ultra-star.de/viewtopic.php?t=2432
And they are CC so as long as attribute the creator and ship a license file with the songs we should be safe.
four songs with covers would be a nice row the new layout 😉 


> Quite don't know if we really need piwic or visitor statistics anyway?

That's the question 😉 It's not much work (guess half an hour) but on the other hand we do not really profit from it (just nice2know info).

**Comment by flokuep** [comment]
> If we move the page to a branch of the main repo it would work without a JEKYLL_GITHUB_TOKEN.
So Vocaluxe/Vocaluxe? Quite don't know if I like this... Seems to be a bit messy, or not?

**Comment by lukeIam** [comment]
>> If we move the page to a branch of the main repo it would work without a JEKYLL_GITHUB_TOKEN.  

> So Vocaluxe/Vocaluxe? Quite don't know if I like this... Seems to be a bit messy, or not?

I agree a separate repo would be nicer - but if we create a brach with no history (`--orphan`) it wouldn't be that messy if it has a meaningful name.

**Comment by flokuep** [comment]
Mhm. Okay. If there is no other way, this should be okay...

**Comment by lukeIam** [comment]
the location has also influence domain options:

`vocaluxe.github.io`:
- `Vocaluxe/Vocaluxe:master` only
- Redirect other domains to it

`vocaluxe.github.io/vocaluxe`:
- `Vocaluxe/Vocaluxe:master` or `Vocaluxe/Vocaluxe:gh-pages`
- Redirect other domains to it

`vocaluxe.github.io/VocaluxeWebsite`:
- `Vocaluxe/VocaluxeWebsite:master` or `Vocaluxe/VocaluxeWebsite:gh-pages`
- Redirect other domains to it

`vocaluxe.org` (use a custom domain):
- `Vocaluxe/VocaluxeWebsite:master` or `Vocaluxe/VocaluxeWebsite:gh-pages` or `Vocaluxe/Vocaluxe:master` or `Vocaluxe/Vocaluxe:gh-pages`
- HTTPS will fail as github pages does not support uploading a custom certificate 
could be workaround with something like [cloudflare's free `Full SSL`](https://blog.cloudflare.com/secure-and-fast-github-pages-with-cloudflare/#step3timeforssl)
- Redirect other domains to it

**Comment by flokuep** [comment]
Okay. There are enough arguments to move it to a branch at this repository. Could you do this?

**Comment by lukeIam** [comment]
> If there is no other way, this should be okay...

There are no private options for github pages 😞  
But I read the documentation again: they are just saying that some fields are not filled without auth.
And locally I get the releases without authentication without any problem... 

> Okay. There are enough arguments to move it to a branch at this repository. Could you do this?

Sure - which one `gh-pages`?

**Comment by flokuep** [comment]
>  And locally I get the releases without authentication without any problem...

Strange... I had to add a key for this to my enivornment variables...

> Sure - which one gh-pages

Yeah. Should be the common one.

**Comment by lukeIam** [comment]
> Sure - which one gh-pages
> Yeah. Should be the common one.

done 

But I have to admit that it is uglier than I thought:
In the repo it looks nice but if you checkout the `gh-pages` branch all the untracked and ignored files from the main project are lying around 😞 

I've pushed it - have a look and decide if we should keep it or delete the branch again...

A solution to work with `gh-pages` on the main repo would be to clone the repo again (maybe with `--single-branch`) to keep both usages separated.

**Comment by lukeIam** [comment]
> Yes, i'm the owner of vocaluxe.org and vocaluxe.net. But only until 17.01.2017 because i have terminated the contracts.

Info:  
`Vocaluxe.org` was transfered to my account.
(configured it to redirect everything to `https://vocaluxe.sourceforge.io`)

Thanks to @brunzel for transferring it.

**Comment by flokuep** [comment]
> Vocaluxe.org was transfered to my account.

Nice! I've commited some of @derivator and @lukeIam suggestions and added a readme with some instructions how to handle this branch when working with it. We might set this branch to protected?

**Comment by lukeIam** [comment]
Just some thoughts:  
- There is no "more" link on the shortened news articles
- The white selections in the main page and the white detailed-news section are looking a bit "unstlyed" (black text on white ground and thats it)
- Building for Linux and getting it running is definitely not easy at the moment. Should we still mention it on the page? Or wait until building is at least a bit easier and/or the required dependencies are better documented?
- The links in the "three-column"-sections are not at the same level. Should we fix that (like in the download section)?
![image](https://cloud.githubusercontent.com/assets/5115160/22079186/31a8a928-ddbb-11e6-8bcb-a99b048e2676.png)

**Comment by hendrikbl** [comment]
I created a website mockup for the new website. I know you already did a lot with the new version and it isn't bad in any way. But I wanted to test out Affinity Designer and used your website as a base :p

So this isn't the complete site and it is just something I did for fun. If you like it, feel free to use it or improve on it. If you don't like it, no problem :)
I'd love to help with the website if you like it, even though I would need a crash course on how to work with Jekyll :p

This is what I've got so far:
![vocaluxe_website](https://cloud.githubusercontent.com/assets/9558391/25234014/23dde5e2-25e1-11e7-8b71-aff8ca88b7b9.png)

**Comment by lukeIam** [comment]
The page vocaluxe.github.io/Vocaluxe/ is not recreated automatically on a metadata change (new nightly).
Is this by design or a bug?
![image](https://user-images.githubusercontent.com/5115160/26868341-fd052ff4-4b68-11e7-8b67-fcea6bfbd268.png)

**Comment by hendrikbl** [comment]
I'm experiencing the same on my fork... It only updates if you push a commit to the gh-pages branch since then it rebuilds the whole page. I'd be surprised though if this is by design...

**Comment by lukeIam** [comment]
> I'd be surprised though if this is by design...

It is - I asked the github support:
```
We're always working to improve GitHub and we consider every suggestion that our amazing users send through to us.

Is there any possibility to trigger a github page recreation on new releases?

This isn't currently possible, but it sounds like it could be pretty useful so I've added your suggestion to our internal Feature Request List.
```

**Comment by lukeIam** [comment]
But I have a workaround: I cloud let the automatic build do a blank commit on the gh-pages branch every time it is pushing a release. (I hope the meta data is updated fast enough).
see #277

**Comment by flokuep** [comment]
Ah, thanks for investigating in this.

**Comment by hendrikbl** [comment]
I had some spare time and tried to realize my mockup i posted a while ago. Here's what I ended up with: https://hendrikbl.github.io/Vocaluxe/

Again, I'd be happy if you like it, but it's not a problem if you don't :)

**Comment by flokuep** [comment]
I like it very much :) I think it's quite better than our current version 👍

**Comment by hendrikbl** [comment]
Okay, glad to hear that :)
I can create a pull request if you want. The changes I made are 99% CSS and contain only a few phrasing changes to the sites content.

**Comment by lukeIam** [comment]
> I can create a pull request if you want.

👍

**Comment by hendrikbl** [comment]
Nice to see the page online! :)

I'm still working on some improvements on the page though. This work contains:

- [ ] RSS Feed
- [ ] Improve responsiveness (different screen sizes/resolutions)
- [ ] Rework news section
  - [ ] Something like #274
  - [ ] Listing/Sorting/Grouping of posts

I'll come back here (or open a pull request) when I'm done. 
And I'm always open for ideas and suggestions! :)

**Comment by lukeIam** [comment]
I see a small problem:
If release a new final version, the displayed beta version is older than that.
-> I suggest to filter out beta releases that have a lower version number than the latest final release.

> Some kind of gallery?

Maybe something linke this:
http://pgwjs.com/pgwslideshow/#example

**Comment by hendrikbl** [comment]
>  I suggest to filter out beta releases that have a lower version number than the latest final release.

You're right, otherwise it could confuse poeple.

> Maybe something linke this:
> http://pgwjs.com/pgwslideshow/#example

I thought about some kind of slideshow, too. I've used [Gallery CSS](http://benschwarz.github.io/gallery-css/) in the past. It's lightweight and easy to use. And since it's pure CSS, it should be fairly easy to incorporate it into the sites design.

**Comment by lukeIam** [comment]
> Maybe something linke this:
> http://pgwjs.com/pgwslideshow/#example
> I thought about some kind of slideshow, too. I've used Gallery CSS in the past. It's lightweight and easy to use. And since it's pure CSS, it should be fairly easy to incorporate it into the sites design.

Were do you plan to use the gallery?
I was thinking about a better way to include images to the news posts.
But if you like to show screenshots or something similar on the main page your solution is better.

An other thing: what about adding a background image to the first selection:
![image](https://user-images.githubusercontent.com/5115160/28212206-9799c814-68a0-11e7-85c7-5b63f8cceae5.png)
```css
    background-image: url(http://www.publicdomainpictures.net/pictures/220000/velka/rock-concert-1494235506Xle.jpg);
    background-size: 115%;
    background-blend-mode: soft-light;
```
and with adjusted title bar:
![image](https://user-images.githubusercontent.com/5115160/28212564-562c302c-68a2-11e7-9c53-80cc6e9fbdbe.png)

**Comment by hendrikbl** [comment]
> I was thinking about a better way to include images to the news posts.

I agree, the current way is pretty ugly i guess... I worked on that a little bit on [my fork](https://hendrikbl.github.io/Vocaluxe/). I think the way they appear in a post is okay, its just the markdown part with adding all the classes and stuff which isn't very nice. 
With that said, what do you think of the layout of the news section?

> But if you like to show screenshots or something similar on the main page your solution is better.

That's what I was thinking about. I always like to see what I get :p
I'll play around with it a bit and see what I can achieve.

> what about adding a background image to the first selection

Looks good, but I think we should change the foreground image. It looks kinda random in my opinion.

> If you scroll to the top the icon in the left top corner is not longer centered (and could be a bit larger in my opinion). See second screen shot.

I'll take a look :)

**Comment by lukeIam** [comment]
> I agree, the current way is pretty ugly i guess... I worked on that a little bit on my fork. I think the way they appear in a post is okay, its just the markdown part with adding all the classes and stuff which isn't very nice.
> With that said, what do you think about the layout of the news section?

👍 I really like the new news overview with the boxes - Nice work!
But you are right the the articles themselves doesn't look very nice (we may need custom css styling here e.g. for the list)


> Looks good, but I think we should change the foreground image. It looks kinda random in my opinion.

Do you mean this image: https://hendrikbl.github.io/Vocaluxe/img/screenshot_sing.png?
But I thing we should show a sing screen here (it's the main window of the application)
Maybe we can also show a song selection screen...

**Comment by hendrikbl** [comment]
> But you are right the the articles themselves doesn't look very nice (we may need custom css styling here e.g. for the list)

That's not exactly what I meant but it's not false either :p
I was talking about the code you have to write to add an image to a posts gallery:
```
[![Song Selection]({{site.baseurl}}/img/posts/release-01_Screenshot_SongSelection.png){:class="img-responsive"}]({{site.baseurl}}/img/posts/release-01_Screenshot_SongSelection.png){:class="img-responsive" data-lightbox="screenshots" data-title="Song Selection"}
```
It would be better to have an array of images in the yaml front matter or something so we can iterate about it when loading the post.

> But I thing we should show a sing screen here (it's the main window of the application)

What do you think about something like this:

![hero_image](https://user-images.githubusercontent.com/9558391/28221015-f490324e-68c1-11e7-8481-932d6c667eea.png)
![hero_solid](https://user-images.githubusercontent.com/9558391/28221019-f8a11a38-68c1-11e7-8973-ddf7881a2121.png)

**Comment by hendrikbl** [comment]
> If you scroll to the top the icon in the left top corner is not longer centered (and could be a bit larger in my opinion). See second screen shot.

This should be fixed now :)

> ![](https://user-images.githubusercontent.com/9558391/28221015-f490324e-68c1-11e7-8481-932d6c667eea.png)

I pushed this version to [my fork](hendrikbl.github.io/Vocaluxe/) along with a test for a gallery. I don't like the gallery very much though. And it makes the navigation very unresponsive with an additional element. I'll propably remove it again...

I also replaced lightbox with [Swipebox](http://brutaldesign.github.io/swipebox/) for a better experience on mobile devices.

**Comment by lukeIam** [comment]
I tested using a custom domain (new.voclauxe.org) successfully.
But as github does not support HTTPS on custom domains it breaks HTTPS support.
We would have to use a https proxy (like the one from cloudflare - i thing it's free).

**Comment by hendrikbl** [comment]
> Edit: Should I try this out?

I read about it and it seems pretty straightforward so I'd say lets try it out. Worst thing to happen is that it doesn't work :p

I'll also try to create a pr in the next days including all the changes I made since the last one.

**Comment by lukeIam** [comment]
> I read about it and it seems pretty straightforward so I'd say lets try it out. Worst thing to happen is that it doesn't work :p

It's live:
https://new.vocaluxe.org

> I'll also try to create a pr in the next days including all the changes I made since the last one.

👍 looking forward to it

**Comment by flokuep** [comment]
Uh. Nice. Thanks for your work +1

**Comment by flokuep** [comment]
Just found a little bug: When showing news articles in details, navigation isn't working anymore (href is just set to #anchor and not to index.html#anchor). It would be nice, if on news sites, _news_ is marked as selected page :)

**Comment by hendrikbl** [comment]
I'm quite busy this week, I'll look into it next week if it isn't fixed until then :)

**Comment by lukeIam** [comment]
@flokuep can you please provide some more details? For me it's working (or
I misunderstood you).


On Aug 15, 2017 08:33, "Hendrik" <notifications@github.com> wrote:

I'm quite busy this week, I'll look into it next week if it isn't fixed
until then :)

—
You are receiving this because you were assigned.

Reply to this email directly, view it on GitHub
<https://github.com/Vocaluxe/Vocaluxe/issues/240#issuecomment-322388295>,
or mute the thread

**Comment by flokuep** [comment]
Just try to go from https://new.vocaluxe.org/2012/12/21/second-release.html to e.g. feature-section. On the same page, it would be nice, if **News** is marked as selected.

**Comment by hendrikbl** [comment]
@flokuep I just tested it on my phone and clicking on __Features__  brings me to the Features section of the index page. That's how it should be I think?

**Comment by lukeIam** [comment]
@flokuep with Firefox and Chrome (desktop and mobile) it's working for
me...

**Comment by flokuep** [comment]
Okay. This gets really strange now. I could _swear_ it doesnt work yesterday. So nevermind.

**Comment by flokuep** [comment]
What about adding a little "First steps" section, that could be openend after installing vocaluxe (#289)? This could be useful for new players.

We could show:
1. How to find songs and put them into vocaluxe -> Song folder in Documents/Vocaluxe for Installer
2. How to setup mics (should be configured automatically, but maybe describe way to records menu)
3. How to add profile
4. Start singing a first song
5. Refer to song managment tool -> uman (https://sourceforge.net/projects/uman/)

...anythin else?

**Comment by lukeIam** [comment]
Just a little addition to point 2:
Configure singstar mics as stereo - default is mono/1 channel (windows)

**Comment by flokuep** [comment]
@lukeIam Good point!

@hendrikbl Do you have time (and desire) to create a small page like this?

**Comment by hendrikbl** [comment]
> Do you have time (and desire) to create a small page like this?

I do. But I don't know how long it will take since I don't have that much time during the week. I'll do my best :)

**Comment by hendrikbl** [comment]
Just thought about it and maybe it's better to create this "First Steps" section in our wiki?
We already have a few HowTos there so it would fit in there.
But if we want a more polished and designed page, I can create one on the website.

**Comment by flokuep** [comment]
I guess a polished and desinged for our website would be nice.

**Comment by hendrikbl** [comment]
> I guess a polished and desinged for our website would be nice.
:+1:

**Comment by lukeIam** [comment]
We may also need to display the installer version (*.exe) if we have any on the page.

**Comment by lukeIam** [comment]
And we have to thing about how to guide the user through Windows' smartscreen filter:
https://github.com/Vocaluxe/Vocaluxe/issues/289#issuecomment-326053900

**Comment by flokuep** [comment]
> We may also need to display the installer version (*.exe) if we have any on the page.

Done with #295

**Comment by lukeIam** [comment]
> We may also need to display the installer version (*.exe) if we have any on the page.
> Done with #295

It's live (https://new.vocaluxe.org) - but the nightly text is now a bit to big:
![image](https://user-images.githubusercontent.com/5115160/29917299-d5653c7a-8e41-11e7-8273-cb17e5830228.png)

**Comment by hendrikbl** [comment]
Yes, it's even worse on mobile...
I have a neat idea how to handle portable and installer versions, I'll try it out later today and let you know :)

**Comment by hendrikbl** [comment]
I committed my first try but the stupid thing won't build... 
You can download and try it if you want or if you know what causes the fail, even better :D

I'll try to fix it later...

**Comment by lukeIam** [comment]
Locally it's building fine...

Any error messages on github?
https://help.github.com/articles/viewing-jekyll-build-error-messages/#viewing-jekyll-build-failure-messages-in-your-repository

I like your idea of the flipping card! great idea!
But I don't really like the drop-downs - how about putting the release name
to the top of the "backside" card and add buttons with os + arch + type
(should be short enough) line by line

**Comment by hendrikbl** [comment]
It only showed a generic error... but it seems to be fixed now. Propably an error in the release query.


> how about putting the release name to the top of the "backside" card and add buttons with is + arch + type (should be short enough) line by line


I did that and I like it! I committed the new version to [my fork](hendrikbl.github.io/Vocaluxe).

**Comment by flokuep** [comment]
Really like this solution, thanks 👍

**Comment by hendrikbl** [comment]
Which branch should I pr to? github-pages or github-pages-installer? :)

**Comment by flokuep** [comment]
gh-pages, gh-pages-installer is already merged (and will be deleted)

**Comment by hendrikbl** [comment]
Okay 👍

**Comment by lukeIam** [comment]
Anything to do here before making this live (voacluxe.org)?

> Missing content? 

-> Post for the new release
(But can be added when the release is ready to ship)


> And we have to thing about how to guide the user through Windows' smartscreen filter:
> #289 (comment)

Some kind of popup after the download how to unblock?
But this isn't mission critical -> we can add this later.

**Comment by lukeIam** [comment]
What need to be done to make the new page live:
- [ ] Update the DNS settings 
- [ ] Update the CNAME file
- [ ] Change the SF page to redirect with `301 Moved Permanently`

**Comment by flokuep** [comment]
I can try to update sf page later!

**Comment by flokuep** [comment]
Okay, everything prepared to change sf page. Do you update dns setting and cname? After that I'll change sf page.

**Comment by lukeIam** [comment]
I will when I'm back at home (~15:00)

**Comment by lukeIam** [comment]
@flokuep 
https://vocaluxe.org is live - you can continue with the migration.

**Comment by flokuep** [comment]
Done.

**Comment by lukeIam** [comment]
> Done.

It's working 😃

**Comment by flokuep** [comment]
Nice :) So merge #294 and then release rc?

**Comment by lukeIam** [comment]
> Nice :) So merge #294 and then release rc?

Do it :)

**Comment by hendrikbl** [comment]
![grafik](https://user-images.githubusercontent.com/9558391/29997392-0a0cc566-9012-11e7-8065-8f2bc4119071.png)

Responsiveness is broken again with more than 3 releases. With linux builds it will be even worse.
Making the cards 'longer' would look strange i think. So what do you think of something simple like this:

![grafik](https://user-images.githubusercontent.com/9558391/29997497-ded65bca-9014-11e7-9591-1e5a70bbc6b2.png)

**Comment by flokuep** [comment]
Ouh. That looks really nice, maybe better than now 👍

**Comment by lukeIam** [comment]
I thought about how people finde USDX and Vocaluxe:
Most likely the google for "singstar" and "pc" -> first page does not contain Vocaluxe (but USDX)

We should thing about adding "singstar" to our description (the visible and the meta one). And about some additional tags for search engine crawlers.

**Comment by hendrikbl** [comment]
Maybe we can use [Google Search Console](https://www.google.com/webmasters/tools/) or/and some other analytics tool to see how people access the website. 

Adding some metadata to it shouldn't be a problem :+1:

**Comment by lukeIam** [comment]
> Maybe we can use Google Search Console

I will give it a try - atm it's crawling data.

> or/and some other analytics tool

We thought about piwik - but decided it's additional work and does not give us real benefits.

BTW: The cloudflare https proxy has also some stats:
![image](https://user-images.githubusercontent.com/5115160/32116271-3dec06cc-bb4a-11e7-947a-b6119d599ce7.png)

![image](https://user-images.githubusercontent.com/5115160/32116298-5c36806c-bb4a-11e7-8872-56209f88b119.png)

**Comment by lukeIam** [comment]
Also found this https://github.com/blog/2162-better-discoverability-for-github-pages-sites

(The last exam for this semester is on the horizon - so I hope I will finde more time for Vocaluxe soon)

**Comment by lukeIam** [comment]
> Maybe we can use Google Search Console
> I will give it a try - atm it's crawling data.

We are only clicked with the search themes `vocaluxe` and `vocalux`.
Sometimes (not often) we are also shown on some other keywords like:
`ultrastar song downlaod` oder `singstar download` but only on position 35 and 89 -> so no clicks here

**Comment by lukeIam** [comment]
As the new website is live and working - I'll close this issue.
Feel free to open a new issue for bugs or new features for the website.
